server/routes/auth.routes.js

```diff
@@ -54,81 +54,64 @@ router.post('/', [
 router.get('/', auth, async(req, res) => {
     try {
         const user = await Employer.findById(req.user.id).select('-password');
-        res.json(user);
+        res.status(200).json(user);
         console.log(user);
     } catch (err) {
         console.error(err.message);
         res.status(500).json({ msg: 'Server Error' });
     }
 });
 
-router.put(
```

Comment:
is there a reason the put / delete functions are taken out from this page  ?

Reply:
I have no idea what happened....Let me review